setup.py

```diff
@@ -112,6 +112,14 @@ def get_extras_require() -> Dict[str, List[str]]:
             "jax",
             "optax",
             "dm-haiku",
+        ]
+        if sys.version_info[:2] < (3, 9)
+        else [
+            "torch==1.7.1 ; sys_platform=='darwin'",
+            "torch==1.7.1+cpu ; sys_platform!='darwin'",
+            "torchvision==0.8.2 ; sys_platform=='darwin'",
+            "torchvision==0.8.2+cpu ; sys_platform!='darwin'",
+            "torchaudio==0.7.2",
```

Comment:
If we manage the dependencies for Python 3.9 separately, the list of software will be doubled in the future. How about adding the version constraints as follows?
```python
        "example": [
            "catboost ; python_version<'3.9'",
            "chainer ; python_version<'3.9'",
            "lightgbm ; python_version<'3.9'",
            "mlflow ; python_version<'3.9'",
            "mpi4py ; python_version<'3.9'",
            "mxnet ; python_version<'3.9'",
            "nbval ; python_version<'3.9'",
            "scikit-image ; python_version<'3.9'",
            "scikit-learn>=0.19.0,<0.23.0 ; python_version<'3.9'",  # optuna/visualization/param_importances.py.
            "xgboost ; python_version<'3.9'",
            "keras ; python_version<'3.9'",
            "tensorflow>=2.0.0 ; python_version<'3.9'",
            "tensorflow-datasets ; python_version<'3.9'",
            "pytorch-ignite ; python_version<'3.9'",
            "pytorch-lightning>=1.0.2 ; python_version<'3.9'",
            "thop ; python_version<'3.9'",
            "skorch ; python_version<'3.9'",
            "stable-baselines3>=0.7.0 ; python_version<'3.9'",
            "catalyst ; python_version<'3.9'",
            "torch==1.7.1 ; sys_platform=='darwin'",
            "torch==1.7.1+cpu ; sys_platform!='darwin'",
            "torchvision==0.8.2 ; sys_platform=='darwin'",
            "torchvision==0.8.2+cpu ; sys_platform!='darwin'",
            "torchaudio==0.7.2",
            "allennlp==1.2.0 ; python_version<'3.9'",
            "dask[dataframe] ; python_version<'3.9'",
            "dask-ml ; python_version<'3.9'",
            "botorch ; python_version>'3.6' and python_version<'3.9'",
            "fastai ; python_version<'3.9'",
            "jax ; python_version<'3.9'",
            "optax ; python_version<'3.9'",
            "dm-haiku ; python_version<'3.9'",
        ],
```

Reply:
I don't think that list whose most items come with the identical constraint among them.

What in my mind is to merge the separate two requirement lists after the Python3.9 list gets the bunch of items.

Reply:
> What in my mind is to merge the separate two requirement lists after the Python3.9 list gets the bunch of items.

I see. Let's merge the lists after we add more packages to the Python 3.9 list.